environment/settings.go

```diff
@@ -108,7 +108,7 @@ func (l Limits) AsContainerResources() container.Resources {
 		MemoryReservation: l.MemoryLimit * 1_000_000,
 		MemorySwap:        l.ConvertedSwap(),
 		BlkioWeight:       l.IoWeight,
-		OomKillDisable:    &l.OOMDisabled,
+		OomKillDisable:    !&l.OOMKiller,
```

Comment:
You could do it like this:

```go

OomKillDisable:    boolPtr(!l.OOMKiller),


// Helper function to create a pointer to a boolean value
func boolPtr(b bool) *bool {
	return &b
}
```

what should do the same, just the other way around as I assume that is what you are trying to do

Reply:
Thanks, I will try that!